<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(48, 48, 48);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_1">
     <item>
      <widget class="QPushButton" name="btn_profile">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Профиль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_favorites">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Избранные игры</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="tableWidget_1">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>50</y>
      <width>670</width>
      <height>540</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>211</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Жанры:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(255, 255, 255);
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>Дата выхода:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(255, 255, 255);
</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
font: 12pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>По оценкам пользователей:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(255, 255, 255);
</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 48, 48);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(48, 48, 48);
font: 10pt &quot;MS Shell Dlg 2&quot;;
font: 12pt &quot;MS Shell Dlg 2&quot;;
font: 10pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Поиск</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
